docs/plugin-development/restrictions.md

```diff
@@ -82,6 +82,67 @@ start debugging performance issues is through the Plugin Statistics window,
 which can be found through Plugins > Open Plugin Stats in the dev menu
 (`/xldev`).
 
+## Can my plugin talk to a backend server I run?
+
+Plugins are permitted to communicate with special backend servers, though there
+are certain considerations and requirements that must be met:
+
+- Plugins should take care to send the minimum amount of data necessary to do
+  their job. Whenever feasible, plugins should hash information like Content IDs
+  or player names on the client side so that a server-side data breach does not
+  reveal information.
+- Plugins may collect additional non-essential information for telemetry,
+  analytics, or statistical purposes, provided the user is given a chance to
+  review what data is being collected and for what purpose.
+  - Users must opt in to additional telemetry collection, but this may be done
+    as part of a "welcome to this plugin" experience forcing an explicit choice,
+    or via a global Dalamud setting.
+  - Additional collected data should be done for the public interest. That is,
+    the extra information should go back to improvement of the plugin, in order
+    to provide the public with statistics, or otherwise improve the game and the
+    experience.
+  - Plugins must use a user-resettable identifier for any non-anonymous
+    analytics data. This identifier may not contain any personal information and
+    must be resettable at any time by the user. Developers are encouraged to
+    take care to prevent analytics information from deanonymizing users, even
+    with full access to the raw dataset.
+- Plugins must take care to not expose a list of other plugin users or allow an
+  easy way to test whether a specific user is using any plugin. Users may list
+  themselves in a public directory if they so choose, but this risk should be
+  identified to the user.
+- Any official backend server for a plugin must use encrypted communication via
+  HTTPS (or an equivalent), and must have certificates issued from a trusted
+  certificate authority such as [Let's Encrypt](https://letsencrypt.org/).
+  Plugins must connect to servers via DNS name to prevent data from going to
+  stray IP addresses.
+
+The appropriateness of data being submitted to backend servers is ultimately
+subjective, and will be handled on a case-by-case basis of the Plugin Approval
+Committee. Factors such as the developer's intent with the data, the necessity
+of the data to collect, and how things are communicated to users will all affect
+what any given plugin would be allowed to collect. For example, there are
+certain classes of data that should _never_ be collected, even with explicit
```

Comment:
Would there be value in eventually having a (noninclusive) list somewhere of which classes of data this is referring to?

Reply:
Removing this entirely as its awkward. Put it this way - nothing I'm envisioning will be normally collected by a user unless they're doing something very very strange. This is another "depending on what PAC things" moment.